targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_WIFI_BT/device/TARGET_MCU_PSOC6_M4/TOOLCHAIN_GCC_ARM/cy8c6xx7_cm4_dual.ld

```diff
@@ -174,6 +174,21 @@ SECTIONS
     } > flash
 
     /* Cortex-M4 application image */
+    /* Places the code in the Execute in Place (XIP) section. See the smif driver
```

Comment:
@hennadiykytsun can you review the changes to the linker script/startup code?

Reply:
Is there a reason we would only change GCC and not the IAR & ARM toolchains?

Reply:
I'd like to get GCC implementation reviewed first before adding IAR and ARM toolchains. It's a time consuming task to get everyone to approve a PR. Doing it incrementally reduces the amount of retesting every time a reviewer's feedback is incorporated.